androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/SeekHandler.kt

```diff
@@ -0,0 +1,153 @@
+package com.profusion.androidenhancedvideoplayer.components.playerOverlay
+
+import androidx.compose.animation.core.FastOutSlowInEasing
+import androidx.compose.animation.core.RepeatMode
+import androidx.compose.animation.core.animateFloat
+import androidx.compose.animation.core.infiniteRepeatable
+import androidx.compose.animation.core.rememberInfiniteTransition
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.media3.exoplayer.ExoPlayer
+import com.profusion.androidenhancedvideoplayer.styling.Colors
+import com.profusion.androidenhancedvideoplayer.utils.DEFAULT_INITIAL_TIME_TO_SEEK
+import com.profusion.androidenhancedvideoplayer.utils.JobsHolder
+import com.profusion.androidenhancedvideoplayer.utils.executeAfterTimeout
+import kotlin.math.max
+import kotlin.math.min
+
+private const val ICON_ANIMATION_DURATION_MS = 650
+private const val ICON_INITIAL_SCALE = 0.8f
+private const val ICON_TARGET_SCALE = 1.1f
+private const val JOB_TIMEOUT = 650L
+private const val TRANSITION_LABEL = "scaleSeekIcon"
+
+@Composable
+fun SeekHandler(
+    exoPlayer: ExoPlayer,
+    setIsControlsVisible: (value: Boolean) -> Unit,
+    transformSeekIncrementRatio: (tapCount: Int) -> Long,
+    toggleIsControlsVisible: () -> Unit,
+    controlsCustomization: ControlsCustomization
+) {
+    val jobs = JobsHolder
+    val scope = rememberCoroutineScope()
+    var forwardTapCount by remember { mutableStateOf(0) }
+    var rewindTapCount by remember { mutableStateOf(0) }
+    val isRewinding by remember { derivedStateOf { rewindTapCount > 0 } }
+    val isForwarding by remember { derivedStateOf { forwardTapCount > 0 } }
+
+    val transition = rememberInfiniteTransition(TRANSITION_LABEL)
+
+    val scale = transition.animateFloat(
```

Comment:
ditto about the label
![image](https://github.com/profusion/android-enhanced-video-player/assets/35314270/d6fcacfb-6cfd-4fbc-a311-0710a682744e)